cmd/gateway/oss/gateway-oss_test.go

```diff
@@ -0,0 +1,177 @@
+package oss
```

Comment:
can you add license header here @timonwong ? just like other gateways?

Reply:
@harshavardhana Fixed